sfcc_cartridge/int_fast_sfra/cartridge/scripts/order/orderHookScripts.js

```diff
@@ -88,11 +88,50 @@ exports.afterPOST = function (order) {
 	
 	//Set the Customer to Order
 	try {
-		var profile = CustomerMgr.queryProfile('email = {0}', order.custom.fastEmailId);
+		var login = order.custom.fastEmailId;
```

Comment:
@purushj , could you create separate PR for new user creation please. We would need to merge quick view and new user items separately

Reply:
@igorzinc , As we discuss and agree to have all the changes in one PR and plan to have new PR for new Changes and we are having different Repos (2 in Born and 1 in Fast) so it will be hard to change all places. Do we have any issues other than the below review comments to have create separate PR?